javascript/computer_science/time_complexity.md

```diff
@@ -14,29 +14,29 @@ In this lesson, we'll introduce the core concepts around measuring the time effi
 
 This section contains a general overview of topics that you will learn in this lesson.
 
-* How the efficiency of an algorithm is measured.
-* What is Big O.
-* What are the Big O notations used to measure an algorithm's efficiency.
-* How else can we measure an algorithm's efficiency.
-* What to do when two algorithms have the same complexity.
+-   How the efficiency of an algorithm is measured.
+-   What is Big O.
+-   What are the Big O notations used to measure an algorithm's efficiency.
+-   How else can we measure an algorithm's efficiency.
+-   What to do when two algorithms have the same complexity.
 
 ### Efficiency basics
 
 The very first step in mastering efficient code is to understand how to measure it. Let's take a look at a simple little program that prints out all odd numbers between 1 and 10.
 
-~~~js
+```js
 function oddNumbersLessThanTen() {
-  let currentNumber = 1;
+	let currentNumber = 1;
 
-  while (currentNumber < 10) {
-    if (currentNumber % 2 !== 0) {
-      console.log(currentNumber);
-    }
+	while (currentNumber < 10) {
+		if (currentNumber % 2 !== 0) {
+			console.log(currentNumber);
+		}
 
-    currentNumber += 1;
-  }
+		currentNumber += 1;
+	}
 }
-~~~
+```
```

Comment:
This needs to go back to 2 space indenting instead of 8. Or these are possibly tabbed indents? In any case, we use 2 spaces to indent inside of codeblocks.